examples/Directory.Build.props

```diff
@@ -1,6 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
     <Import Project="$(MSBuildThisFileDirectory)../build/IceRpc.Examples.props" />
+    <Import Project="$(MSBuildThisFileDirectory)../build/IceRpc.Version.props" />
```

Comment:
This import the Version variable that MSBuild uses to assign a version for the assemblies it builds.